lib/definitions/build.d.ts

```diff
@@ -17,12 +17,15 @@ interface IiOSBuildData extends IBuildData {
 	iCloudContainerEnvironment: string;
 }
 
-interface IAndroidBuildData extends IBuildData {
+interface IAndroidBuildData extends IBuildData, IAndroidSigningData {
```

Comment:
You can try using `IHasAndroidBundle`. There the flag is optional, but you can change it to mandatory and just extend with `Partial<IHasAndroidBundle>` wherever needed.